ts/features/bonus/common/BaseTimeoutScreen.tsx

```diff
@@ -1,12 +1,12 @@
 import * as React from "react";
 import { SafeAreaView } from "react-native";
-import image from "../../../../../img/wallet/errors/payment-expired-icon.png";
-import { renderInfoRasterImage } from "../../../../components/infoScreen/imageRendering";
-import { InfoScreenComponent } from "../../../../components/infoScreen/InfoScreenComponent";
-import I18n from "../../../../i18n";
-import { cancelButtonProps } from "./buttons/ButtonConfigurations";
-import { FooterStackButton } from "./buttons/FooterStackButtons";
-import { bonusVacanzeStyle } from "./Styles";
+import image from "../../../../img/wallet/errors/payment-expired-icon.png";
+import { renderInfoRasterImage } from "../../../components/infoScreen/imageRendering";
+import { InfoScreenComponent } from "../../../components/infoScreen/InfoScreenComponent";
+import I18n from "../../../i18n";
+import { cancelButtonProps } from "../bonusVacanze/components/buttons/ButtonConfigurations";
+import { FooterStackButton } from "../bonusVacanze/components/buttons/FooterStackButtons";
```

Comment:
I guess this two components may be moved in  common folder too, they've been reused inside the app

Reply:
Yes it is part of another PR because it is not common only to bonuses but also outside.

Reply:
Ok, so we can merge this and then migrate these components in another PR